modules/cnv-about-fencing.adoc

```diff
@@ -0,0 +1,41 @@
+// Module included in the following assemblies:
+//
+// * cnv/cnv_users_guide/cnv-fencing-nodes.adoc
+
+[id="cnv-about-fencing_{context}"]
+
+= About fencing nodes
+
+Fencing is the act of protecting shared resources in order to safely recover 
+from node failure.
+
+In the case of an unhealthy node, Pods and virtual machines that depend on 
+persistent storage cannot be replaced on another node until it can be confirmed 
+that the original Pod or virtual machine is no longer accessing the storage. 
+Fencing the node ensures that the workloads are no longer running on the 
+unhealthy node, protecting the persistent storage. Once fencing is complete, 
+workloads can be safely replaced on other nodes.
+
+The fencing feature introduces three new controllers, which are deployed by the 
+`MachineRemediation` Operator:
+
+* `MachineDisruptionBudget` controller
+* `MachineHealthCheck` controller
+* `MachineRemediation` controller
+
+The `MachineDisruptionBudget` controller creates a `MachineDisruptionBudget` object 
```

Comment:
Technically the `MachineDisruptionBudget` objects are created by the admin. There is also a default one created by the MachineRemediationOperator